src/Compilers/CSharp/Portable/Symbols/PublicNullableAnnotationExtensions.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Diagnostics;
+
+namespace Microsoft.CodeAnalysis.CSharp
+{
+    internal static class PublicNullableAnnotationExtensions
```

Comment:
>PublicNullableAnnotationExtensions [](start = 26, length = 34)

Consider moving this file to the root of the project and moving the class to the `CodeAnalysis` namespace. Then the `Public` prefix is not needed.